woocommerce/woocommerce
# Add $item to wc_downloadable_file_permission

### All Submissions:


### Changes proposed in this Pull Request:
Adding the parameter `$item` to both the function `wc_downloadable_file_permission` and the filter hook `woocommerce_downloadable_file_permission` would provide a further level of customization for the downloadable file being granted, depending on the metadata assigned to the single item when purchasing a product.

### How to test the changes in this Pull Request:

1. add a filter to the hook `woocommerce_downloadable_file_permission`, including 5 parameters
2. print the 5th parameter `$item` to verify that it is passed the data of the single item in each order

### Other information:


### Changelog entry
Added parameter `$item` (instance of WC_Order_Item) to both the function `wc_downloadable_file_permission` and the filter hook `woocommerce_downloadable_file_permission`
